assets/dev/js/editor/elements/manager.js

```diff
@@ -18,38 +19,31 @@ export default class ElementsManager {
 	 * Function getElementType().
 	 *
 	 * Get element type by elType, and widgetType, if exact widgetType is not found
-	 * the method will return the element type ( WidgetBase ).
+	 * the method will return the element type.
 	 *
 	 * @param {string} elType
 	 * @param {string|false} widgetType
 	 */
 	getElementType( elType, widgetType = false ) {
```

Comment:
Try to use the widget as an element
```suggestion
	getElementType( elType ) {
```

Reply:
Handled.

Reply:
Handled.